src/components/input/inputText/inputText.tsx

```diff
@@ -0,0 +1,14 @@
+import { InputContainer, type IInputComponentProps } from '../inputContainer';
+import { useInputProps } from '../useInputProps';
+
+export interface IInputTextProps extends Omit<IInputComponentProps, 'type'> {}
```

Comment:
Is there a case of an input for which we *won't* want to omit the type?

Reply:
Mm.. actually no, we will always set a static `type` property for all the inputs. It makes sense to apply this for all the inputs 👌🏻